CMakeLists.txt

```diff
@@ -97,6 +97,9 @@ endif ()
 #
 #####################################################################################################################
 
+# for rapidjson folder in ssc
+include_directories($ENV{SSCDIR})
```

Comment:
Would a environment variable RAPIDJSONDIR work? For SAM developers, this could be set to the SSC directory. Developers without SSC could set it to some other directory with rapidjson. (this might be a good item for the meeting agenda tomorrow)

Reply:
> Would an environment variable RAPIDJSONDIR work? For SAM developers, this could be set to the SSC directory. Developers without SSC could set it to some other directory with rapidjson. (this might be a good item for the meeting agenda tomorrow)

Updated and working in wex, SAM and SAM-private.